### DISCUSSION TITLE
Tutorial on jupyterlab-myst + MyST markdown

### DISCUSSION THREAD

**Discussion by mmcky** [QuantEcon/reading-group-2024#8] [Polls]
We would like to organise a brief training session on [jupyterlab-myst](https://manual.quantecon.org/writing/jupyterlab-myst.html#) covering:

1. Installation
2. What is `MyST` markdown
3. reading `md` and `ipynb` files in jupyterlab

Some [further resources are available here](https://github.com/QuantEcon/reading-group-2024/discussions/6)

**Expected Duration: 20-30min**

**Comment by mmcky** [comment]
@pgrosser1 @mbek0605 @Jiarui-ZH if you haven't responded yet -- if you could let me know your time preferences that would be great. If need be I can host two separate sessions (Tue and Wed). Thanks.

**Reply by mbek0605** [reply]
I am available on Tuesday and Wednesday after 09:30 pm EST/01:30 pm Canberra time.

**Reply by mmcky** [reply]
@mbek0605 I can meet up either today at 1:30pm or tomorrow at 1:30pm. Let me know which works best for you.

**Reply by pgrosser1** [reply]
I can do anytime tomorrow if that works for you Matt? @mmcky

**Reply by Jiarui-ZH** [reply]
I can do anytime tomorrow after 1:30pm Canberra time @mmcky

**Comment by mmcky** [comment]
Perhaps the easiest solution will be to meet up and have a zoom session with you individually. I know @Jiarui-ZH is travelling today.

**Comment by mmcky** [comment]
Thanks @pgrosser1 @mbek0605 and @Jiarui-ZH -- I will run a session tomorrow (Wednesday) @ 1:30pm Canberra time. 

If you can't make that time we can do a 1-on-1 session.

Thanks.

**Reply by mbek0605** [reply]
Certainly! Apologies, but could you please share a Zoom link? I do not think I received one for today's meeting.

**Reply by mmcky** [reply]
I have just sent this link out via email.

**Comment by mmcky** [comment]
Thanks @pgrosser1 @Jiarui-ZH @mbek0605 -- here is a link to manual page re: `jupyterlab-myst`

https://manual.quantecon.org/writing/jupyterlab-myst.html
